cmd/jujud/agent/machine.go

```diff
@@ -1923,10 +1929,8 @@ func (a *MachineAgent) uninstallAgent(agentConfig agent.Config) error {
 
 	errors = append(errors, a.removeJujudSymlinks()...)
 
-	insideLXC, err := lxcutils.RunningInsideLXC()
-	if err != nil {
-		errors = append(errors, err)
-	} else if insideLXC {
+	insideLXC := container.RunningInContainer()
```

Comment:
Probably we should switch to "insideContainer" as the variable name for clarity.